assessment_software/src/app/TextField-Component/TextField.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input, Output, EventEmitter } from "@angular/core";
+
+@Component({
+    selector:'app-textfield',
+    templateUrl:'./TextField.component.html',
+    styleUrls:['./TextField.component.css'],
+    
+})
+
+export class TextField{
+    // test:any;
```

Comment:
There is a comment and unused code. Since, on approval of this pull request, the branch will be merged to the main branch, it is best to remove unnecessary comments.

Reply:
@Aben-Bel removed the comments